types/collections.go

```diff
@@ -33,6 +33,9 @@ var (
 	// IntValue represents a collections.ValueCodec to work with Int.
 	IntValue collcodec.ValueCodec[math.Int] = intValueCodec{}
 
+	// UintValue represents a collections.ValueCodec to work with Uint.
+	UintValue collcodec.ValueCodec[math.Uint] = uintValueCodec{}
```

Comment:
Please do add a test using: https://github.com/cosmos/cosmos-sdk/blob/b027112d0a6aa2f2660164b6c00d743bac963f72/collections/colltest/codec.go#L51

🙇‍♂️

Reply:
added to `types/codec_test.go`